code/nebula/neblightning.h

```diff
@@ -45,6 +45,7 @@ typedef struct l_bolt {
 	int		delay;				// delay stamp
 	int		strikes_left;		// #of strikes left
 	float	width;
+	bool	playSound;
```

Comment:
Pedantic, I know, but should be snake case. Ditto for the parameter names of the same, etc.

Reply:
Done